src/main/frontend/format/block.cljs

```diff
@@ -257,6 +257,26 @@
               body)]
     (assoc block :body body)))
 
+(defn extract-filetags
+  [content]
+
+  (def filetags-prefix "#+filetags: ")
+
+  (defn extract-line-filetags
```

Comment:
probably longer than it should be, happy for suggestions :) e.g. maybe could use if-let instead somehow or something similar?

Reply:
the code looks great, but I think you don't have to use regex to match the filetags, because all directives will be collected at https://github.com/logseq/logseq/blob/master/src/main/frontend/format/mldoc.cljs#L80, you might need to parse the `filetags` and merge to the `:page/tags` because those tags are pages themselves.
Something like this:
``` clojure
alias (->vec-concat (:roam_alias properties) (:alias properties))
          tags (->vec-concat (:roam_tags properties) (:tags properties) definition-tags)
          org-file-tags (:filetags properties)
          tags (if org-file-tags
                 (let [file-tags (->> (string/split org-file-tags ":")
                                      (remove string/blank?))]
                   (->vec-concat tags file-tags))
                 tags)
```

Reply:
ah excellent! This `collect-page-proprties` is exactly what I was looking for, not sure how I missed it since I was grepping for `:title` to figure out where it's parsed :)

Reply:
Sorry for the lacking documentation, feel free to ping me on discord anytime.

Reply:
No worries. It's hard to document such details, especially when everything is changing all the time.